### PR TITLE
[fix](invert index) ensure that the pred result sign of the inlist is…

### PR DESCRIPTION
**Pull request by zzzxl1993**
… in order

## Proposed changes

1. If the pred_result_sign generated by inlist with inverted index is out of order, the result will be wrong

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/olap/rowset/segment_v2/segment_iterator.h [hunk 1]
```diff
@@ -101,7 +101,7 @@ struct ColumnPredicateInfo {
     }
 
     std::string column_name;
-    std::vector<std::string> query_values;
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/segment_iterator.h:104]: add document to explain using set

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]

**Pull request merged**